go.mod

```
@@ -0,0 +1,3 @@
+module microservice-framework
```

It's not a framework, but an app

```suggestion
module microservice-app
```